laravel/resources/views/schedules/resident/schedule_confirm.blade.php

```diff
@@ -1,202 +1,66 @@
-	<form method="POST" action="./submit">
-		<input type="hidden" name="schedule_id" value="{{ $id }}">
+	<!-- Use a hidden form to store selected schedule ids, milestones, education objectives, and anest pref. -->
+<form method="POST" action="./submit">
+	<input type="hidden" name="schedule_id" value="{{ $id }}">
 	<div class="row">
-		@if(!is_null($previous[0]))
+		@if(!is_null($previousChoices[0]))
+			<div class="col-12 col-md-6">
+				<h3>Your Previous Selections</h3>
+			</div>
+		@endif
 		<div class="col-12 col-md-6">
-			<h3>Your Previous Selections</h3>
-			<h4>First Choice: </h4>
-			<p>Date: {{ $previous[0]['schedule'][0]['date'] }}</p>
-			<p>Room: {{ $previous[0]['schedule'][0]['room'] }}</p>
-			<p>Case Procedures:<br><?php
-				$case_procedure = $previous[0]['schedule'][0]['case_procedure'];
-				$case_procedure = preg_replace('/[0-9]+/', '', $case_procedure);
-				$case_procedure = preg_replace('/[:\/]/', '', $case_procedure);
-				$case_procedure = preg_replace('/\(|\)/', '', $case_procedure);
-				$case_procedure = str_replace(' [','',$case_procedure);
-				$case_procedure = str_replace(array('[',']'),'',$case_procedure);
-				echo nl2br($case_procedure);
-			?>
-			</p>
-		    <p>Milestone:<br> {{ $previous[0]['milestone'][0]['category'] }} - {{$previous[0]['milestone'][0]['detail'] }}</p>
-		    <p>Objective:<br> {{ $previous[0]['prevPref'][0]['objectives'] }}</p>
-			@if ($previous[0]['pref_anest'] != '[]') 
-				<p>Preferred Anesthesiologist: <br> {{$previous[0]['pref_anest'][0]['first_name']}} {{$previous[0]['pref_anest'][0]['last_name']}}</p>
-				@else 
-				<p>Preferred Anesthesiologist: <br> No Preference </p>
-				@endif
-							
-
-
-			<h4>Second Choice: </h4>
-		    @if(is_null($previous[1]))
-				<p>None</p>
-			@else
-				<p>Date: {{ $previous[1]['schedule'][0]['date'] }}</p>
-				<p>Room: {{ $previous[1]['schedule'][0]['room'] }}</p>
-				<p>Case Procedures:<br><?php
-					$case_procedure = $previous[1]['schedule'][0]['case_procedure'];
-					$case_procedure = preg_replace('/[0-9]+/', '', $case_procedure);
-					$case_procedure = preg_replace('/[:\/]/', '', $case_procedure);
-					$case_procedure = preg_replace('/\(|\)/', '', $case_procedure);
-					$case_procedure = str_replace(' [','',$case_procedure);
-					$case_procedure = str_replace(array('[',']'),'',$case_procedure);
-					echo nl2br($case_procedure);
-				?>
-				</p>
-			    <p>Milestone:<br> {{ $previous[1]['milestone'][0]['category'] }} - {{$previous[1]['milestone'][0]['detail'] }}</p>
-			    <p>Objective:<br> {{ $previous[1]['prevPref'][0]['objectives'] }}</p>
-				@if ($previous[1]['pref_anest'] != '[]')
-				<p>Preferred Anesthesiologist: <br> {{$previous[1]['pref_anest'][0]['first_name']}} {{$previous[1]['pref_anest'][0]['last_name']}}</p>
-				@else 
-				<p>Preferred Anesthesiologist: <br> No Preference </p>
-				@endif
+			<h3>Your Current Selections</h3>
+		</div>
+	</div>
+	@for($i = 0; $i < 3; $i++)
```

Comment:
rather than looping 3 times, is it possible to use a foreach?

Consider if in the future, REMODEL needs to take a different number of options. Obviously, this would be a **massive** change, but avoiding the "magic number" here might make such a change slightly easier.

Reply:
The previous layout logic uses a for each loop (1st commit). If we decide to do the layout update, I will change to foreach loop otherwise I will leave it like this until confirmation.

Reply:
Sounds good! I prefer the visuals of pairing up the different choices in the redesign (and the foreach loop!).